src/psyclone/tests/parse/kernel_test.py

```diff
@@ -65,6 +68,75 @@
 
     )
 
+# Code fragment for testing kernel which uses an interface
+# instead of a type-bound procedure
+CODE_INTERFACE = (
+    "module test_mod\n"
+    "  type, extends(kernel_type) :: test_type\n"
+    "    type(arg_type), dimension(1) :: meta_args =    &\n"
+    "          (/ arg_type(gh_field,gh_inc,w1) /)\n"
+    "     integer :: iterates_over = cells\n"
+    "   contains\n"
+    "  end type test_type\n"
+    "  interface test_code\n"
+    "    module procedure sub_code\n"
+    "  end interface test_code\n"
+    "contains\n"
+    "  subroutine sub_code()\n"
+    "  end subroutine sub_code\n"
+    "end module test_mod\n"
+
+    )
+
+# Code fragment for (failure) test for kernel with two
+# interfaces and no type-bound procedure.
+CODE_DOUBLE_INTERFACE = (
+    "module test_mod\n"
+    "  type, extends(kernel_type) :: test_type\n"
+    "    type(arg_type), dimension(1) :: meta_args =    &\n"
+    "          (/ arg_type(gh_field,gh_inc,w1) /)\n"
+    "     integer :: iterates_over = cells\n"
+    "   contains\n"
+    "  end type test_type\n"
+    "  interface test_code\n"
+    "    module procedure sub_code\n"
+    "  end interface test_code\n"
+    "  interface test_code_again\n"
+    "    module procedure more_code\n"
+    "  end interface test_code_again\n"
+    "contains\n"
+    "  subroutine sub_code()\n"
+    "  end subroutine sub_code\n"
+    "end module test_mod\n"
+
+    )
+
+CODE_DOUBLE_PROCEDURE = (
+    "module test_mod\n"
+    "  type, extends(kernel_type) :: test_type\n"
+    "    type(arg_type), dimension(1) :: meta_args =    &\n"
+    "          (/ arg_type(gh_field,gh_inc,w1) /)\n"
+    "     integer :: iterates_over = cells\n"
+    "   contains\n"
+    "  end type test_type\n"
+    "  interface test_code\n"
+    "    module procedure sub_code, more_code\n"
+    "  end interface test_code\n"
+    "contains\n"
+    "  subroutine sub_code()\n"
+    "  end subroutine sub_code\n"
+    "  subroutine more_code()\n"
+    "  end subroutine more_code\n"
+    "end module test_mod\n"
+
+    )
+
+
+@pytest.fixture(scope="module", params=[CODE, CODE_INTERFACE])
```

Comment:
In order to fix some pylint complaints, please could you add "name='get_code_fragment'" here and then rename the function itself below to e.g. get_code_fragment_fixture.

Reply:
done.